packages/now-static-build/src/utils/gatsby-config.ts

```diff
@@ -0,0 +1,80 @@
+import { writeFile, copyFile, mkdir } from 'fs';
+import { join, dirname } from 'path';
+import { promisify } from 'util';
+import { isCanary } from './is-canary';
+
+const writeFilePromise = promisify(writeFile);
+const mkdirPromise = promisify(mkdir);
+const copyFilePromise = promisify(copyFile);
+
+const GATSBY_PLUGIN_NOW_NAME = 'gatsby-plugin-now';
+const GATSBY_USER_CONFIG_PATH = './__now_gatsby_config_user.js';
+
+function createGatsbyConfig(hasUserConfig: boolean) {
+  return `let userConfig = {}
+
+${hasUserConfig ? `userConfig = require('${GATSBY_USER_CONFIG_PATH}')` : ''}
```

Comment:
```suggestion
let userConfig = ${hasUserConfig ? `require('${GATSBY_USER_CONFIG_PATH}')` : '{}'}
```
You can do this with one line instead of two